pkg/patterns/addon/pkg/status/kstatus.go

```diff
@@ -44,21 +44,26 @@ func (k *kstatusAggregator) BuildStatus(ctx context.Context, info *declarative.S
 		}
 	}
 
+	// Abnormal-true refers to the present state condition where something unusual happens.
+	// Here we augment each deployment manifests abnormal-true conditions and determine the declarativeObject's present
+	// condition from these conditions.
+	// https://github.com/kubernetes-sigs/cli-utils/tree/master/pkg/kstatus#conditions
+	var totalAbnormalTrueConditions []status.Condition
+
 	if shouldComputeHealthFromObjects {
 		statusMap := make(map[status.Status]bool)
 		for _, object := range info.Manifest.Items {
 			gvk := object.GroupVersionKind()
 			nn := object.NamespacedName()
 
-			log := log.WithValues("kind", gvk.Kind).WithValues("name", nn.Name).WithValues("namespace", nn.Namespace)
+			log.WithValues("kind", gvk.Kind).WithValues("name", nn.Name).WithValues("namespace", nn.Namespace)
```

Comment:
So this line doesn't actually log (IIUC).  It sets up a log context which includes the kind and namesapce and name in any messages that are logged.  It means we don't need to repeat this information in each of the log messages below.  So I think we do want `log := log...` (as before)